kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyInitializer.java

```diff
@@ -271,4 +283,17 @@ public void selectServer(NetFilter.NetFilterContext context) {
             context.initiateConnect(target, filters);
         }
     }
+
+    @Sharable
+    private static class ErrorHandler extends ChannelInboundHandlerAdapter {
+        @Override
+        public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
+            if (LOGGER.isDebugEnabled()) {
+                LOGGER.warn("An exceptionCaught() event was caught by the error handler: {}", cause.getMessage(), cause);
+            }
+            else {
+                LOGGER.warn("An exceptionCaught() event was caught by the error handler: {}", cause.getMessage());
```

Comment:
Should we advise in the message that you can get the full stack trace by raising the logging level to DEBUG